api/EncodedVideoChunk.json

```diff
@@ -162,7 +170,7 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": false
+              "version_added": "preview"
```

Comment:
Should this also have the partial implementation and note?

Reply:
Sure :-)
```suggestion
              "version_added": "preview",
              "partial_implementation": true,
              "notes": "Only supported on Linux, see <a href='https://bugzil.la/1749047'>bug 1749047</a>."
```